lib/modules/manager/github-actions/extract.spec.ts

```diff
@@ -1,6 +1,29 @@
 import { Fixtures } from '../../../../test/fixtures';
 import { extractPackageFile } from '.';
 
+const runnerTestWorkflow = `
```

Comment:
What's the behavior when someone uses this syntax?
(Renovate can't update it, but should fail horribly either 🤓)
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-combining-groups-and-labels

Reply:
Nice catch! I have added a test case to confirm that such use cases are ignored.